package.json

```diff
@@ -1,16 +1,69 @@
 {
 	"name": "@adobe/aio-cli-plugin-api-mesh",
-	"version": "3.2.0",
-	"publishConfig": {
-		"access": "public"
+	"version": "3.1.0",
+	"description": "Adobe I/O CLI plugin to develop and manage API mesh sources",
+	"keywords": [
+		"oclif-plugin"
+	],
+	"homepage": "https://www.npmjs.com/package/@adobe/aio-cli-plugin-api-mesh",
+	"bugs": {
+		"url": "https://github.com/adobe/aio-cli-plugin-api-mesh/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/adobe/aio-cli-plugin-api-mesh"
+	},
+	"license": "Apache-2.0",
+	"author": "Adobe Inc.",
+	"main": "src/index.js",
+	"directories": {
+		"test": "test"
+	},
+	"files": [
+		"/oclif.manifest.json",
+		"/src"
+	],
+	"scripts": {
+		"e2e": "jest --collectCoverage=false --testRegex './e2e/e2e.js'",
+		"format": "prettier --check \"./**/*.{ts,tsx,js,css,json,md}\"",
+		"format:fix": "prettier --write  \"./**/*.{ts,tsx,js,css,json,md}\"",
+		"lint": "eslint .",
+		"lint:fix": "eslint --fix .",
+		"prepack": "oclif-dev manifest && oclif-dev readme",
+		"postpack": "rm -f oclif.manifest.json",
+		"test": "jest",
+		"test:ci": "jest --ci",
+		"unit-tests": "jest --ci",
+		"version": "oclif-dev readme && git add README.md"
 	},
 	"dependencies": {
-		"@adobe-apimesh/mesh-builder": "1.3.2",
+		"@adobe-apimesh/mesh-builder": "1.4.0-beta.6",
```

Comment:
same question regarding going to develop with beta version

Reply:
This was for testing, I published beta packages for testing. Once the PRs are approved, we can publish stable packages and update here.